### ISSUE TITLE
Angaben im Benutzerkonto sprachlich anpassen

### ISSUE THREAD

**Issue by relhei** [ubtue/tuefind#2507]
Im Benutzerkonto gibt es noch Inkonsistenzen bei den Bezeichnungen:
Z. B. Spracheinstellung Deutsch und Französisch:
Die Auswahl der Bezeichnungen unter "Anrede" ist in beiden Fällen noch auf Englisch (Ms. bzw. Mr.). Ist sicher nicht ganz so wichtig, aber vielleicht könnte das noch bei Gelegenheit analog zur Angabe "Land" angepasst, also entsprechend der Sprachauswahl der Benutzeroberfläche angezeigt werden.
![grafik](https://github.com/ubtue/tuefind/assets/25769591/26266043-0ba1-4ae9-85cd-fa11e02b7a74)
Auf der deutschen Oberfläche sollte es auch heißen "Meine persönlichen Angaben" statt jetzt:
![grafik](https://github.com/ubtue/tuefind/assets/25769591/be8ebd25-33fc-4f35-8269-a1c8efcf14c8)
Analog dazu auch im Englischen: aktuell "Your profile" sollte ersetzt werden durch "My profile".
![grafik](https://github.com/ubtue/tuefind/assets/25769591/f54c3624-841b-4f6d-939a-3f9afa8c6269)
Auf der französischen Benutzeroberfläche dominiert noch "votre" etc., das an mehreren Stellen durch "mon" etc. ersetzt werden müsste.
![grafik](https://github.com/ubtue/tuefind/assets/25769591/bf4c1feb-b706-41f5-aa27-0205d5718573)

**Comment by mtrojan-ub** [comment]
Ich schau's mir bei Gelegenheit an. Es kann sein, dass wir Mr/Ms. usw mal selbst hinzugefügt hatten und das ebenfalls lokalisiert werden müsste. Bei den anderen Dingen wäre ja auch interessant ob vufind-org hier in sicht konsistent ist oder nicht, ich muss also prüfen ob manche Dinge an die Community weitergereicht werden müssen.

**Comment by relhei** [comment]
Ja, das Thema müsste man für alle Sprachen (bei IxTheo sind es ja noch mehr) mal konzertiert angehen. Unstimmigkeiten gibt es z. B. auch bei "My  Newsfeeds" und "My publications" > beide aktuell nur auf Deutsch und Englisch (alle anderen Sprachen auf Englisch, nur bei Französisch gibt es eine kleine Abweichung) verfügbar. Von VuFind Seite komisch erscheint mir auch, dass Dt. "Anrede" auf Englisch und in allen anderen Sprachen "Appellation" heißt. Das kann ich mir nicht so richtig vorstellen, dass das stimmt (v.a. bei den nicht-lateinischen Schriften ist es auffällig).
